test/common/TestBedChild.cpp

```diff
@@ -473,16 +487,30 @@ namespace RcclUnitTesting
       }
     }
 
+    #ifdef ENABLE_OPENMP
+    int numThreadsToUse = this->useRankThreading ? numRanksToExecute : 1;
+    #endif
+
     // Start group call
     CHILD_NCCL_CALL(ncclGroupStart(), "ncclGroupStart");
 
     // Loop over all collectives to be executed in group call
     for (int collId = 0; collId < this->numCollectivesInGroup[groupId]; ++collId)
     {
       // Loop over all local ranks
+      ErrCode errCode = TEST_SUCCESS;
+      #ifdef ENABLE_OPENMP
+      if (this->verbose) INFO("Group %d collective %d running %d threads\n", groupId, collId, numThreadsToUse);
+      auto& errCodeVal = reinterpret_cast<int&>(errCode);
+      #pragma omp parallel for num_threads(numThreadsToUse) reduction(max : errCodeVal)
+      #endif
```

Comment:
I don't think you actually need to worry about #ifdef-ing this out.
The pragma will just get ignored if we didn't compile for OpenMP.

Reply:
I'd prefer the `INFO` not to print, but I can just key that off of `this->useRankThreading`?

Reply:
Yes - I think so.